source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,49 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template<typename Policy, typename InputKeyIt, typename InputValueIt,
+        typename Comparator =
+            std::less<typename std::iterator_traits<InputIt>::value_type>>
+    void
+    sort_by_key(Policy&& policy, InputKeyIt keys_first, InputKeyIt keys_last,
+                InputValueIt values_first, Comparator comp =
+                    std::less<typename std::iterator_traits<InputKeyIt>::value_type>());
+
+``oneapi::dpl::sort_by_key`` sorts a sequence of keys within the range ``[keys_first, keys_last)``.
+It also permutes the corresponding sequence of values within the range
+``[values_first, values_first + distance(keys_first, keys_last))``.
+Each key is associated with a corresponding value,
+and this association is maintained throughout the sorting process.
+
+The order of equal keys is not guaranteed to be preserved.
+
+``comp`` is a function object,
+which satisfies the ``Compare`` named requirement defined by the `C++ Standard`_.
```

Comment:
```suggestion
``comp`` is a function object that satisfies the requirements defined by the `C++ Standard`_
for the equivalent parameter of `std::sort`.
```
That's because it is arguable if `Compare` is really defined as a named requirement in the standard. Also, explicitly referring to `std::sort` might be useful.

Reply:
Applied for both `sort_by_key` and `stable_sort_by_key`.